README.md

```diff
@@ -1,63 +1,94 @@
-<div align="center">
-  <img src="mussie.png" alt="logo" width="300px"  height="300px" />
-  <br/>
-  <h3><b>Webpack App Starter</b></h3>
+# LeaderBoard Project
+
+<a name="readme-top"></a>
+
+  <h3><b>LeaderBoard Project COmpelet</b></h3>
+
 </div>
 
-This is a simple starter project for building web applications using Webpack. It includes configuration files for development and production builds, as well as basic file structure for a typical web app.
 
-## Getting Started
+# 📗 Table of Contents
+
+- [LeaderBoard Project](#leaderboard-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[LeaderBoard Project\] ](#-leaderboard-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [👥 Authors ](#-authors-)
+  - [📝 License ](#-license-)
+
```

Comment:
- [ ] In order not to block you I'll review your project, however, All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo. in your readme file the following parts are still missing:
  - Getting Started
  - Future Features 
  - Contributing 
  - Show your support 
  - Acknowledgments